test/algorithms/test_grover_optimizer.py

```diff
@@ -40,6 +41,7 @@
 class TestGroverOptimizer(QiskitOptimizationTestCase):
     """GroverOptimizer tests."""
 
+    @unittest.skipUnless(optionals.HAS_AER, "qiskit-aer is required to run this test")
```

Comment:
At some point, since the reference Sampler is probably fast enough, we could probably change tests like these so we do not require Aer any longer and this test no longer is an optional one - though CI of course will install Aer. I would leave things as they are now. Its more just a comment.

Reply:
Sorry that I removed this line accidentally in #542. I intended to revert the removal in this PR. It requires aer sampler because terra sampler takes much more time than aer sampler (#451 for details).